### ISSUE TITLE
Grouping field types

### ISSUE THREAD

**Issue by abemedia** [dobtco/formbuilder#65]
Currently it's only possible to have 2 types: inputs & non-inputs, however it would be great to be able to have an unlimited number of groups which would allow keeping it tidy when adding lots of field types by putting them into accordions or something similar.

**Comment by ajb** [comment]
Yep, totally agree. If you want to help build this, I'm more than happy to help review/accept PRs.
